scm/etc/scripts/ccpp_suite_sim/plt_scmout_2d.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+#################################################################################################
+# Dependencies
+#################################################################################################
+import os
+import xarray as xr
+import matplotlib.pyplot as plt
+import argparse as ap
+
+#################################################################################################
+def read_SCMout2d(fileIN,vars2plt):
+    #
+    data = xr.open_dataset(fileIN)
+
+    # Get list of available variables.
+    varsavail = list(data.keys())
+
+    # Check that requested variables (vars2plt) exist.
+    for var2plt in vars2plt:
+        if (not var2plt in varsavail):
+            print('ERROR: ',var2plt, ' is not in ',fileIN,'.')
+            exit()
+
+    # Read in data, store in python dictonary "state", return.
+    state={}
+    state["time"] = data["time_inst"].values/3600.  # sec -> hour
+    state["pres"] = data["pres"].values[:,:,0]*0.01 # Pa -> hPa
+    for var2plt in vars2plt:
+        state[var2plt]          = data[var2plt]
+        state[var2plt+"_units"] = data[var2plt].units
+        state[var2plt+"_time"]  = data[data[var2plt].dims[0][0:len(data[var2plt].dims[0])-4]].values/3600.
+
+    #
+    return (state)
+
+#################################################################################################
+# Argument list
+# ./plt_scmout_2d.py -n twpice -sdf SCM_GFS_v17_p8 -nmls input_GFS_v17_p8_simA input_GFS_v17_p8_simB -vars sfc_dwn_sw sfc_up_sw sfc_dwn_lw
+#################################################################################################
+parser = ap.ArgumentParser()
+parser.add_argument('-n',    '--case_name', help='name of case', required=True)
+parser.add_argument('-sdf',  '--suite',     help='CCPP suite definition file',required=True)
+parser.add_argument('-nmls', '--nml_list',  help='namelists, separated by a space', nargs='*', required=True)
+parser.add_argument('-vars', '--var_list',  help='varaibles to plot, separated by a space', nargs='*', required=True)
+
+def main():
+
+    #############################################################################################
+    # Get arguments
+    #############################################################################################
+    args      = parser.parse_args()
+    case_name = args.case_name
+    suite     = args.suite
+    namelist  = args.nml_list
+    vars2plt  = args.var_list
+
+    #############################################################################################
+    # Get data
+    #############################################################################################
+
+    # Get files to plot
+    nfiles  = 0
+    fileSCM = []
+    for nml in namelist:
+        fileSCM.append("../../../../scm/run/output_" + case_name + "_" + suite+"_"+nml+"/output.nc")
```

Comment:
This does not work for my test runs as the 'nml' is not appended to my case directory.
For example, when executing this script, it looks for 
../../../../scm/run/output_MAGIC_LEG15A_SCM_GFS_v17_p8_input_GFS_v17_p8/output.nc  does not exist.
But it is named:
../../../run/output_MAGIC_LEG15A_SCM_GFS_v17_p8/output.nc

It looks like the 3d works in this way as well.

Reply:
@hertneky Good catch. It should be fixed now.